.github/workflows/release.yml

```diff
@@ -65,3 +67,12 @@ jobs:
           GIT_COMMITTER_NAME: getsentry-bot
           GIT_AUTHOR_NAME: getsentry-bot
           EMAIL: bot@getsentry.com
+      - id: bump-license-date
+        if: ${{ !github.event.inputs.dry_run && !github.event.inputs.version }}
+        env:
+          GIT_COMMITTER_NAME: getsentry-bot
+          GIT_AUTHOR_NAME: getsentry-bot
+          EMAIL: bot@getsentry.com
+        run: |
+          sed -i -e "s/\(Change Date:\s*\)[-0-9]\+\$/\\1$(date +'%Y-%m-%d' -d '3 years')/" LICENSE
```

Comment:
Instead of repeating this logic from the `bump-version.sh`, we can also do `git checkout <release_branch> -- LICENSE`. I was being lazy